uber/templates/fields/attendee.html

```diff
@@ -1,7 +1,7 @@
 {% import 'macros.html' as macros with context %}
 
 {#- Setting some variables to keep the size of conditional clauses smaller. -#}
-{%- set is_prereg_confirm_email_enabled = (is_prereg_attendee or is_prereg_dealer) and c.PREREG_CONFIRM_EMAIL_ENABLED -%}
```

Comment:
I was running through testing the new modular fields and found it kind of weird that we're only asking people to confirm their email if they're on the pre-reg form -- badge transfers and group members filling out their info only had to enter their email once, so I've changed the condition. We still won't require an email twice on the admin form, that check is in the field itself instead of here.